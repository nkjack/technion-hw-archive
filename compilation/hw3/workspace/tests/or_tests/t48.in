void main(){
byte c = 2500b;
}
