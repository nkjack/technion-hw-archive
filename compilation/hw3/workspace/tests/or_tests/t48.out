line 2: byte value 2500 out of range
